website/docs/best-practices/how-we-mesh/mesh-3-structures.md

```diff
@@ -42,15 +48,37 @@ Horizontal splits separate your DAG based on source or domain. These splits are
 
 A multi-project architecture can exist in a single repo (monorepo) or as multiple projects, with each one being in their own repository (multi-repo).
 
 - If you're a **smaller team** looking primarily to speed up and simplify development, a **monorepo** is likely the right choice, but can become unwieldy as the number of projects, models and contributors grow.
 - If you’re a **larger team with multiple groups**, and need to decouple projects for security and enablement of different development styles and rhythms, a **multi-repo setup** is your best bet.
+
+## Projects, splits, and teams
+
+Since the laumch of dbt Mesh, the most common pattern we've seen is one where **projects are 1:1 aligned to teams**, and each project is its own codebase in its own repository. This isn’t a hard-and-fast rule: Some organizations want multiple teams working out of a monorepo. Some teams own multiple domains that feel awkward to keep combined.
+
+Users may need to contribute models across multiple projects. This is fine! There will be some more friction to do this, versus the monorepo — but this is **useful** friction, especially if upstreaming a change from a “spoke” to a “hub.” This should be treated like making an API change, one that the other team will be living with for some time to come. You should be concerned if your teammates find they need to make a coordinated change across multiple projects very frequently (every week), or as a key prerequisite for ~20%+ of their work.
+
+### Tips and tricks
+
+In the next page, we'll go through a more in depth example of how to split a monolithic project into multiple projects. Here are some tips to get you started when starting to think about using the splitting methods listed above on your own projects:
+
+1. Start by drawing a diagram of your teams doing data work. Map each team to a single dbt project. If you already have an existing monolithic project, and you’re onboarding *net-new teams,* this could be as simple as declaring the existing project as your “Hub” and creating new “Spoke” sandbox projects for each of those teams.
+2. **Split off common foundations** when ****you know that multiple downstream teams will require the same data source. Those could be upstreamed into a centralized Hub, or split off into a separate foundational project. need some splits to facilitate other splits, e.g. source staging models in A that are used in both B + C (lack of project cycles).
+3. **Split again** to ~~introduce intentional friction /~~ encapsulate a particular set of models (e.g. for external export).
+4. **Recombine** if you have “hot path” subsets of the DAG that you need to deploy with low latency, because it powers in-app reporting or operational analytics. It might make sense to have a different dedicated team own these data models (see principle 1), similar to how software services with significantly different performance characteristics often warrant dedicated infrastructure, architecture, and staffing.
```

Comment:
```suggestion
4. Recombine if you have “hot path” subsets of the DAG that you need to deploy with low latency because it powers in-app reporting or operational analytics. It might make sense to have a different dedicated team own these data models (see principle 1), similar to how software services with significantly different performance characteristics often warrant dedicated infrastructure, architecture, and staffing.
```